src/qt/peertablemodel.h

```diff
@@ -66,6 +66,10 @@ class PeerTableModel : public QAbstractTableModel
         Subversion = 5
     };
 
+    enum {
+        StatsRole = Qt::UserRole
+    };
```

Comment:
644f5f5 I'm not very familiar with the gui code, but this enum seems an odd way to do this. Maybe an explanatory comment would be good.

Reply:
As an alternative, a simple `constexpr` could be used (it seems better for me), but here I use the way how we already did such things:
https://github.com/bitcoin-core/gui/blob/f1f26b8d5baec4a45a3a9ba0440cd4eff7af8407/src/qt/addresstablemodel.h#L36-L38

https://github.com/bitcoin-core/gui/blob/f1f26b8d5baec4a45a3a9ba0440cd4eff7af8407/src/qt/bitcoinunits.h#L91-L94

https://github.com/bitcoin-core/gui/blob/f1f26b8d5baec4a45a3a9ba0440cd4eff7af8407/src/qt/coincontroldialog.h#L82-L86

https://github.com/bitcoin-core/gui/blob/f1f26b8d5baec4a45a3a9ba0440cd4eff7af8407/src/qt/transactiontablemodel.h#L46

@promag What do you think?

Reply:
Right, this follow Qt approach, I guess they didn't want to manually assign a value to each role.

With multiple roles `constexpr` looks weird because you'd have to manually increment for each custom role.

Reply:
@promag 
> ... this follow Qt approach...

In the [recent example](https://doc.qt.io/qt-5.12/qtwidgets-painting-basicdrawing-example.html) they use
```
const int IdRole = Qt::UserRole;
```


> With multiple roles `constexpr` looks weird because you'd have to manually increment for each custom role.

Agree.

Also https://stackoverflow.com/questions/899917/why-do-people-use-enums-in-c-as-constants-while-they-can-use-const

Reply:
Thanks :)